site/tutorials/tutorial-two-java.md

```diff
@@ -463,7 +464,7 @@ And our `Worker.java`:
             }
           }
         };
-        channel.basicConsume(TASK_QUEUE_NAME, false, consumer);
+        channel.basicConsume(TASK_QUEUE_NAME, consumer);
```

Comment:
Original documentation above says "It's time to remove this flag". Should I change that as well?